doc/en/components/grids/_shared/row-drag.md

```diff
@@ -316,6 +316,55 @@ The drag handle icon can be templated using the grid's `DragIndicatorIconTemplat
 
 To do so, we can use the `DragIndicatorIcon` to pass a template inside of the `{ComponentSelector}`'s body:
 
+<!-- ComponentStart: HierarchicalGrid -->
+
+```tsx
+    function dragIndicatorIconTemplate(ctx: IgrGridEmptyTemplateContext) {
+        return (
+            <>
+                <IgrIcon name="drag_handle" collection="material" />
+            </>
+        );
+    }
+
+    <IgrHierarchicalGrid rowDraggable="true" dragIndicatorIcon={dragIndicatorIconTemplate}>
+    </IgrHierarchicalGrid>
+```
+
+```razor
```

Comment:
This template doesn't seem to render for Blazor. Wrapping just this code snippet in both in Blazor and inner HierarchicalGrid comment tags seems to work. The rest should be unaffected except maybe combine them around to wrap them in a single HierarchicalGrid tag.